python_scripts/linear_models_sol_01.py

```diff
@@ -0,0 +1,133 @@
+# %% [markdown]
+# # Solution for Exercise 01
+#
+# The aim of this exercise is to get intuitions regarding the parameters of a
+# simple linear model and how to quantify the error of such model.
+#
+# We will reuse part of the code of the course to:
+# * load data;
+# * create the function representing a linear model;
+# * plot the data and the linear model function.
+#
+# ## Prerequisite
+#
+# ### Data loading
+
+# %%
+import pandas as pd
+
+data = pd.read_csv("../datasets/penguins.csv")
+# select the features of interest
+feature_names = "Flipper Length (mm)"
+target_name = "Body Mass (g)"
+X = data[[feature_names]].dropna()
```

Comment:
Should we be using another dataset, without na, rather than having to deal with NAs.

Reply:
Since this is something that we are storing locally, we could always have a version of CSV without the NA.